SYCL/GroupAlgorithm/inclusive_scan_sycl2020.cpp

```diff
@@ -0,0 +1,148 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I . -o %t.out
```

Comment:
Please apply all comments from `exclusive_scan_sycl2020.cpp` to this test too.

Reply:
Changes from exclusive_scan_sycl2020.cpp applied to inclusive_scan_sycl2020.cpp